contributor_docs/project_wrapups/anaisgonzalez_gsoc_2021.md

```diff
@@ -0,0 +1,105 @@
+
+![immersive-360-gif](https://user-images.githubusercontent.com/83731139/130161902-dbcda386-b55b-4454-8101-f108aa47e89d.gif)
+
+PRs in this section: [#145](https://github.com/stalgiag/p5.xr/pull/145) , [#146](https://github.com/stalgiag/p5.xr/pull/146)
+
+
+
+### [Example #4: Physics](https://github.com/stalgiag/p5.xr/tree/master/examples/physics)
+I had never worked on a physics example before, so I tried watching some Coding Train tutorials on strings, but couldn't get that to function correctly in VR. After speaking with my mentor about it, they showed me a working physics example that I was able to expand upon for the complex version of this theme. The basic example includes boundaries and a ball that can be held and thrown around.  
+
+![physics-gif-1](https://user-images.githubusercontent.com/83731139/130161918-e969e700-eea6-46dc-923a-95979775cd6e.gif)
+
+For the complex example, I made the Ball class from earlier generate multiple balls at random x, y, and z locations that could change size, shape, texture, and color the moment they'd collide with a boundary. I also tried to include type textures on the shapes too, but they didn't display correctly for some reason, so instead I displayed the type textures on the boundaries of the room. I eventually removed the ability for the ball to change shape or texture since it felt too busy and just left it so it changes only size and color upon collision. Once I added in the other walls and ceiling, the whole thing really came together.  
+
+![physics-gif-2](https://user-images.githubusercontent.com/83731139/130161920-9fea2aa4-8e75-458a-9bac-97e67b9ce463.gif)
+
+PRs in this section: [#143](https://github.com/stalgiag/p5.xr/pull/143) , [#144](https://github.com/stalgiag/p5.xr/pull/144)
+
+
+
+### [Example #5: Embodiment](https://github.com/stalgiag/p5.xr/pull/147)
+For the embodiment example, my mentor explained some other XR functions to me that helped me try to position things in VR. We can get the location of the camera with viewerPosition and we can also get the pose of the camera with viewerPoseMatrix. We can use "applyMatrix(viewerPoseMatrix);" on the head of the body, which allows it mirror the direction/pose that the viewers head is moving. By putting viewerPosition inside of translate, now the other parts of the body will be relative to the location of the head.
+
+![embodiment-gif-1](https://user-images.githubusercontent.com/83731139/130161883-66469af7-3e6b-4ab8-bd11-ef92933a9402.gif)
+
+I didn't get to finish the complex example because I ran out of time. I wanted to create a dragon that the viewer could look at and move with, but I was having trouble converting the size of the dragon's body in WEBGL to the correct dimensions in VR, which are extremely small whenever hand positioning is included. I only managed to get the head and wings working so far.
+
+PRs in this section: [#147](https://github.com/stalgiag/p5.xr/pull/147)
+
+
+
+## Work Pull Requests and Issues
```

Comment:
I would move this section to the top so that readers can check this out first if they wish.